compliance/tests/selector_expr_lhs_multi_assign/selector_expr_lhs_multi_assign.gs.ts

```diff
@@ -53,8 +53,13 @@ function getCoords(): [number, number] {
 }
 
 export function main(): void {
-	let p: Point = null
+	let p: Point = new Point()
 	// This assignment should trigger the error because p.X and p.Y are *ast.SelectorExpr,
 	// and the current implementation of writeMultiVarAssignFromCall in WriteStmtAssign
 	// expects *ast.Ident for destructuring.
-	[
\ No newline at end of file
+	const _tmp = getCoords()
+	p.X = _tmp[0]
```

Comment:
Update the compiler to wrap this part in {} so that it has a scope so we don't have to worry about _tmp being declared multiple times.